install.sh

```diff
@@ -445,11 +465,145 @@ EOF
 
 deploy_volume_provisioner() {
     local data_dir=$1
+    local docker_mirror=$2
     local config_file=${data_dir}/local-volume-provisionser.yaml
-    local config_url="https://raw.githubusercontent.com/pingcap/chaos-mesh/master/manifests/local-volume-provisioner.yaml"
 
-    rm -rf "${config_file}"
-    ensure curl -o "${config_file}" "$config_url"
+    volume_provisioner_image="quay.io/external_storage/local-volume-provisioner:v2.3.2"
+    if [ "$docker_mirror" == "true" ]; then
+        azk8spull volume_provisioner_image || true
+        kind load docker-image ${volume_provisioner_image} > /dev/null 2>&1 || true
+    fi
+
+    cat <<EOF >"${config_file}"
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: "local-storage"
+provisioner: "kubernetes.io/no-provisioner"
+volumeBindingMode: "WaitForFirstConsumer"
```

Comment:
Why we install local-pv here ?

Reply:
this is just an available option for developer